vsutil/__init__.py

```diff
@@ -279,3 +272,56 @@ def _resolve_enum(enum: Type[E], value: Any, var_name: str, module: Optional[str
         return enum(value)
     except ValueError:
         raise ValueError(f'{var_name} must be in {_readable_enums(enum, module)}.') from None
+
+
+def _should_dither(out_bits: int,
```

Comment:
You converted a 4-line condition into a 50 line helper function. This is way overkill.

Reply:
Now 15 loc, most of this is the params not fitting on one line and the explanation that was originally in-line comments.